app.py

```diff
@@ -279,7 +296,7 @@ def _launch_app_internal(self, context, file_to_open=None, version=None):
 
         # get the app args:
         platform_name = {"linux2": "linux", "darwin": "mac", "win32": "windows"}[sys.platform]
```

Comment:
With this change, I don't think we need the `platform_name` anymore in this scope.